rapids-cmake/cpm/versions.json

```diff
@@ -1,6 +1,11 @@
 
 {
   "packages" : {
+    "benchmark" : {
+      "version" : "1.8.0",
+      "git_url" : "https://github.com/google/benchmark.git",
+      "git_tag" : "v${1.8.0}"
```

Comment:
```suggestion
      "git_tag" : "v${version}"
```

Reply:
Sorry I think I got caught somewhere in the middle of the undo chain when reordering as requested in the last review...